README.md

```diff
@@ -186,6 +188,8 @@ author:
     comment:
 melody:
 composer:
+notes:
+  -
```

Comment:
I don't think this would not be good practice for yaml

Reply:
Same with below

Reply:
I'm a bit worried since it would pass a test when written like this. But I realize it's good to keep it like this, but the test should be changed to verify that notes are either undefined or is an array where all elements are of type string